src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Performance/RecommendCaseInsensitiveStringComparison.Base.Tests.cs

```diff
@@ -7,15 +7,25 @@ namespace Microsoft.NetCore.Analyzers.Performance.UnitTests
 {
     public abstract class RecommendCaseInsensitiveStringComparison_Base_Tests
     {
-        private static readonly Tuple<string, string>[] Cultures = new[] {
-            Tuple.Create("ToLower", "CurrentCultureIgnoreCase"),
-            Tuple.Create("ToUpper", "CurrentCultureIgnoreCase"),
-            Tuple.Create("ToLowerInvariant", "InvariantCultureIgnoreCase"),
-            Tuple.Create("ToUpperInvariant", "InvariantCultureIgnoreCase")
+        private static readonly ValueTuple<string, string>[] Cultures = new[] {
+            ValueTuple.Create("ToLower", "CurrentCultureIgnoreCase"),
+            ValueTuple.Create("ToUpper", "CurrentCultureIgnoreCase"),
+            ValueTuple.Create("ToLowerInvariant", "InvariantCultureIgnoreCase"),
+            ValueTuple.Create("ToUpperInvariant", "InvariantCultureIgnoreCase")
```

Comment:
```suggestion
        private static readonly (string, string)[] Cultures = new[] {
            ("ToLower", "CurrentCultureIgnoreCase"),
            ("ToUpper", "CurrentCultureIgnoreCase"),
            ("ToLowerInvariant", "InvariantCultureIgnoreCase"),
            ("ToUpperInvariant", "InvariantCultureIgnoreCase")
```

Reply:
I like it. But it makes me sad that we don't have that analyzer enabled in this repo, I didn't get the automatic suggestion.